app/controllers/events/view/export.js

```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  eventExportStatus : '',
+  eventDownloadUrl  : '',
+  actions           : {
+    startGeneration() {
+      let payload = {
+        image    : this.get('model.exportData.dataImage'),
+        video    : this.get('model.exportData.dataVideo'),
+        audio    : this.get('model.exportData.dataAudio'),
+        document : this.get('model.exportData.dataDocument')
+      };
+      let exportUrl = `/events/${this.get('model').currentEventID}/export/json`;
+      let that = this;
```

Comment:
Don't do this. Use [arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) instead.